docs/running.rst

```diff
@@ -14,10 +14,23 @@ the site.
 
 The recommended ways to run Wooey are:
 
+Through two separate processes
+------------------------------
+
+You can run Wooey by calling two commands (you will need a
+separate process for each):
+
+::
+
+    python manage.py celery worker -c 1 --beat -l info
+    python manage.py runserver
+
+On Windows, remove the ``--beat`` option.
```

Comment:
Can you rephrase this to say under Windows the beat option may not be supported. If you encounter an error, omit this argument.  I'd rather handle cases where in the future it may work.